blocks/header/header.css

```diff
@@ -150,6 +150,7 @@ header nav .nav-sections ul {
 
 header nav .nav-sections picture {
   margin-right: 6px;
+  float: left;
```

Comment:
please use flex instead of float

Reply:
`float:` can be removed. No need for flexboxes... ok ?